File: backend/src/main/java/ipeps/pwd/wallet/payload/createPayload/ContactCreatePayload.java
package ipeps.pwd.wallet.payload.createPayload;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ContactCreatePayload {
    private String firstname;
    private String lastname;
    private String email;
    private String phone;
}

File: frontend/src/app/modules/contact/model/payload/contact.create.payload.ts
import {PayloadInterface} from "@shared/model";

//payload = donnée transmise pour créer contact
export interface ContactCreatePayload extends PayloadInterface{
  lastname: string;
  firstname: string;
  email: string;
  phone: string;
  address: string;

  deleted:boolean;
  deletedAt:Date;
  deletedBy:string;
}
